packages/nextjs-mf/src/plugins/container/InvertedContainerPlugin.ts

```diff
@@ -1,4 +1,4 @@
-import { Compiler } from 'webpack';
+import type { Compiler } from 'webpack';
```

Comment:
Should this use the normalize webpack path here?

Reply:
Type import is fine, those get stripped, only imports that go to src code in build